interview-questions.md

```diff
@@ -8,36 +8,47 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer:  To fix this mistake and properly connect the two tables we would need to create a migration that adds a column (cohort_id) to the students table. So, the foreign key will be named cohort_id and it will be in the Student model
 
-Researched answer:
+Researched answer: The foreign key always goes on the belongs_to side,  which in this case it is the Student model. Foreign keys should always be an integer and be the modelname_id, which here is cohort_id.
```

Comment:
Both answers are good, however you could add content explain how to go about doing this.